XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -127,16 +127,36 @@ A set of flags indicating properties or other options associated with this **`MP
 | `lsfMPTCanTrade`    | `0x0010`   | If set, indicates that _individual_ holders can trade their balances using the XRP Ledger DEX or AMM.                                                                                                                                     |
 | `lsfMPTCanTransfer` | ️`0x0020`  | If set, indicates that tokens held by non-issuers may be transferred to other accounts. If not set, indicates that tokens held by non-issuers may not be transferred except back to the issuer; this enables use-cases like store credit. |
 | `lsfMPTCanClawback` | ️`0x0040`  | If set, indicates that the issuer may use the `Clawback` transaction to clawback value from _individual_ holders.                                                                                                                         |
-
-Except for `lsfMPTLocked`, which can be mutated via the `**MPTokenIssuanceSet**` transactions, these flags are **immutable**: they can only be set during the **`MPTokenIssuanceCreate`** transaction and cannot be changed later.
+Except for `lsfMPTLocked`, which can be mutated via the **`MPTokenIssuanceSet`** transaction, these flags are
```

Comment:
Need an empty line between the table and this paragraph.
![image](https://github.com/user-attachments/assets/c5709713-67f2-4f39-9434-d20c92094505)

Reply:
Nice catch - fixed via 9ebb5dd57890e697431cc62acfaf6ac5722375ee